src/org/thoughtcrime/securesms/preferences/AppProtectionPreferenceFragment.java

```diff
@@ -174,18 +210,6 @@ public void onClick(DialogInterface dialog, int which) {
     }
   }
 
-  private class ReadReceiptToggleListener implements Preference.OnPreferenceChangeListener {
```

Comment:
Maybe the naming of this is throwing me off... is this something that has to do with your feature, or did it accidentally get deleted?

Reply:
It somehow got duplicated. So I deleted one instance of it. If you check the full code, one copy of it is still there :)